test/controllers/ControllerBaseSpec.scala

```diff
@@ -16,12 +16,12 @@
 
 package controllers
 
-import mocks.{MockAuditingService, MockAuth, MockHttp}
+import mocks.{MockAuth, MockHttp}
 import play.api.http.Status
 import play.api.mvc.{Action, AnyContent}
 import play.api.test.Injecting
 
-trait ControllerBaseSpec extends MockAuth with MockHttp with MockAuditingService with Injecting {
+trait ControllerBaseSpec extends MockAuth with Injecting {
```

Comment:
My previous comment was more saying that `MockHttp` is unused so we might as well delete it altogether. Can you also remove the import above.